apps/condo/domains/organization/hooks/useFilteredB2BAppsData.ts

```
@@ -0,0 +1,114 @@
+import get from 'lodash/get'
+import uniqBy from 'lodash/uniqBy'
+import { useMemo } from 'react'
```

ESLint

[error] There should be no empty line within import group

(import/order)